frontend/app/views/fragments/tier/benefits.scala.html

```diff
@@ -55,9 +55,15 @@ <h2 class="tier-header__title">
                     <a class="action js-opt-become-member" href="@routes.Joiner.enterDetails(tier)" id="qa-@tier.slug" data-link-name="Become a @tier">Become a @tier</a>
                     @if(showLearnMore) {
                         @if(tier == Tier.Patron) {
```

Comment:
the only difference here is the href. Could this be just be surrounded by the if statement? (makes scales hand movement)